### PR TITLE
Ruff: comprehensions conventions

### PR DESCRIPTION
**Pull request by jnewb1**

### CODE REVIEW

File: system/sensord/tests/test_sensord.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def test_lsm6ds3_timing(self):
   def test_events_check(self):
     # verify if all sensors produce events
 
-    sensor_events = dict()
+    sensor_events = {}
```
Comment (adeebshihadeh) [inline system/sensord/tests/test_sensord.py:173]: is there a reasoning for this one? rest of the changes look good

Comment (jnewb1) [reply system/sensord/tests/test_sensord.py:173]: C408: rewrite dict() as {}
https://pypi.org/project/flake8-comprehensions/

Comment (adeebshihadeh) [reply system/sensord/tests/test_sensord.py:173]: hmm, their reasoning is that it's slower. don't think it's enough of a difference to restrict the style, so let's ignore C408

Comment (jnewb1) [reply system/sensord/tests/test_sensord.py]: done

**Review by adeebshihadeh — APPROVED** [review]

**Pull request merged**